internal/pkg/template/templates/task/cf.yml

```diff
@@ -255,7 +256,48 @@ Resources:
     Properties:
       LogGroupName: !Join ['', ["/copilot/", !Ref TaskName]]
       RetentionInDays: !Ref LogRetention
+  S3Bucket:
```

Comment:
maybe i'm not quite piecing everything together from the different threads, so if you could help summarize that would be great😅: why create a new bucket instead of using the app bucket?

Reply:
ok actually i think i figured it out - you can do `copilot task run` without having created an app :)

Reply:
so follow-up question which i think is true, but just to verify: running `task delete` will
1. empty the s3 bucket
2. delete the cf stack, which will delete the s3 bucket

Reply:
That's all correct I think!